### ISSUE TITLE
improve formatting of brave://settings/help info

### ISSUE THREAD

**Issue by bradleyrichter** [brave/brave-browser#2560]
## old: (seems confusing, sounds like this is the Chromium version)

![image](https://user-images.githubusercontent.com/13509546/49974024-4e0cd180-feec-11e8-94db-a9cbff2945ce.png)

## new (build should match this format 😄)

![image](https://user-images.githubusercontent.com/13509546/49973988-29185e80-feec-11e8-9408-64dba95004a6.png)

Basically, move "Chromium to a second line.
Move and reorder the "official build dev bits".

**Comment by bsclifton** [comment]
Fix done with https://github.com/brave/brave-core/pull/4480 now reverted with https://github.com/brave/brave-core/pull/5258

cc: @jonathanKingston

**Comment by xxKeefer** [comment]
looking to contribute for my first time, but i'm not sure what needs to be done. how can i help?

**Comment by prithvi009** [comment]
hey @bradleyrichter, is this issue still open?

**Comment by bradleyrichter** [comment]
Yes. This is open and still not fixed. This is a good one for any first contribution commit if @xxKeefer or @prithvi009 wants to take it?

**Comment by spiritanand** [comment]
Has this issue been resolved? Can I work on it?

**Comment by benjaminknox** [comment]
I believe I have a fix for it, let me know what you guys think: https://github.com/brave/brave-core/pull/20528

**Comment by bsclifton** [comment]
Tested with first Nightly version w/ this fix. Looks great!

![Image](https://github.com/user-attachments/assets/873217ea-a860-42da-8c2a-c8bb002de5d3)

**Comment by btlechowski** [comment]
Probably introduced a regression: https://github.com/brave/brave-browser/issues/45840
